lib/activerecord-import/import.rb

```diff
@@ -389,6 +389,8 @@ def import_helper( *args )
       elsif args.last.is_a?( Array ) && args.last.empty?
         return ActiveRecord::Import::Result.new([], 0, [])
         # supports 2-element array and array
+      elsif args.size == 2 && args.first.is_a?( Array ) && args.last.is_a?( Array ) && !args.last.first.is_a?(Array)
+        raise ArgumentError, "last argument should be a two dimensional array '[[]]'. First element in array was a #{args.last.first.class}"
       elsif args.size == 2 && args.first.is_a?( Array ) && args.last.is_a?( Array )
         column_names, array_of_attributes = args
```

Comment:
I think I would rather have the argument check here-ish (line 393). Something like:

```
unless args.last.first.is_a?(Array)
  raise ArgumentError, "Last argument should be a two dimensional array '[[]]'. First element in array was a #{args.last.first.class}"
end
```